Tests/Block/PageListBlockServiceTest.php

```diff
@@ -29,7 +29,7 @@ protected function setUp()
     {
         parent::setUp();
 
-        $this->pageManager = $this->getMock('Sonata\PageBundle\Model\PageManagerInterface');
+        $this->pageManager = $this->createMock('Sonata\PageBundle\Model\PageManagerInterface');
```

Comment:
Oh wait, it does't have it on new versions either, it just has its own `createMock`